qa/integration/specs/pipeline_log_spec.rb

```diff
@@ -133,11 +160,11 @@
   end
 
   @private
```

Comment:
unrelated, but worth cleaning up.

It looks like this is meant to use `Module#private`, which causes subsequent `def`'s in the current scope to be defined privately. As-is, it simply references the `@private` ivar and ignores the result.

Since this is in an rspec example group, we don't really have the _concept_ of public and private methods; methods defined using `def` are only available to examples within the group, so they are _effectively_-private without any further action.

```suggestion
```